pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -30,6 +31,7 @@ import (
 
 const (
 	machineAnnotationKey = "machine.openshift.io/machine"
+	annotationReboot     = "machineremediation.openshift.io/reboot"
```

Comment:
`machineremediation.healthchecking.openshift.io`?

Reply:
`s/annotationReboot/machineRebootAnnotationKey`

Reply:
Given `healthchecking.openshift.io` is the CRD group name I would prefer to still keep `healthchecking.openshift.io` prefix without subgrouping it into `machineremediation`. If any, you might go with `healthchecking.openshift.io/machine-remediation-reboot` or similar.

Reply:
done